thefuck/rules/php_s.py

```diff
@@ -0,0 +1,9 @@
+from thefuck.utils import replace_argument
+
+
+def match(command):
+    return "php -s" in command.script
+
+
+def get_new_command(command):
+    return replace_argument(command.script, "-s", "-S")
```

Comment:
`-s` (lowercase) is a valid command and will timeout when ran by TheFuck. Meaning output will be empty. The rule must tell it doesn't require output. So, add this to the end of the rule:

```python
requires_output = False
```